skimage/feature/peak.py

```diff
@@ -41,6 +42,8 @@ def peak_local_max(image, min_distance=1, threshold_abs=None,
     num_peaks : int, optional
         Maximum number of peaks. When the number of peaks exceeds `num_peaks`,
         return `num_peaks` peaks based on highest peak intensity.
+    num_peaks_per_label : int, optional
+        Maximum number of peaks for each label.
```

Comment:
Good point. I hesitated because it looks less natural, but you're right. We can change that later... As time goes, I'm more and more favourable to move to py3 only (I was not clear even few months ago).